plenum/test/helper.py

```diff
@@ -1060,10 +1061,10 @@ def sdk_check_reply(req_res):
         raise AssertionError("Got an error with code {} for request {}"
```

Comment:
Parse sdk error code and create specific type for important runtime errors

Reply:
Done